Cargo.toml

```diff
@@ -137,6 +137,8 @@ resvg = "0.35"
 rustc-hash = "1.0"
 smol = "1.0"
 softbuffer = "0.2"
+strum = "0.25"
```

Comment:
why adding those dependencies?

Reply:
By its nature, `ComboBox` is meant for selection of an item. When the set of choices is fixed, it's natural to represent the choices with an enum. The [`strum`](https://crates.io/crates/strum) crate has over 53M downloads and in this case it's used to get an iterator over enum variants. I suspect that it's a very common use case.

Maybe the dependency should be pushed to the `Cargo.toml` of the example but my concern is that keeping it updated can be problematic.

Reply:
I know exactly what those things do and I use them myself
no matter how many downloads it has imo there shouldn't be any dependencies added to the crate that aren't needed just because they seem useful to you.